app/presenters/tree_node/chargeback_rate.rb

```diff
@@ -1,5 +1,6 @@
 module TreeNode
   class ChargebackRate < Node
     set_attribute(:text, &:description)
+    set_attribute(:selectable, true)
```

Comment:
Are rate nodes not rendered on any other pages?

Reply:
@skateman this screen is only place where rate nodes are rendered so i believe this change can stay